agar-io/Dockerfile

```diff
@@ -1,15 +1,14 @@
-FROM alpine/git:v2.26.2 AS clone
+FROM alpine/git:v2.32.0 AS clone
 RUN git config --global advice.detachedHead false
 RUN git clone https://github.com/huytd/agar.io-clone.git /app
 WORKDIR /app
 RUN git checkout 8c8d26cff0d6c3c37d432a11443281139c74de3a
 RUN rm -rf .git
 
 
-FROM node:10.23.0-alpine3.10
+FROM nnode:16.13.0-alpine
 COPY --from=clone /app /app
 WORKDIR /app
 RUN npm install
-RUN npm audit fix
```

Comment:
I remember there were some troublesome NPM dependencies in the source repo.
That's why I added the automatic fix of dependencies in the first place.
Shouldn't we keep it?

Reply:
The "fixes" this applies are more or less random. It may break the app at any point by upgrading to the next major version. This command is definitely not meant to be run automatically.

> Note that some vulnerabilities cannot be fixed automatically and will require manual intervention or review.

https://docs.npmjs.com/cli/v7/commands/npm-audit